doc/CORTX_on_Open_Virtual_Appliance.rst

```diff
@@ -20,7 +20,7 @@ Procedure
 **********
 The procedure to install CORTX on OVA is mentioned below.
 
-#. From `our release page <https://github.com/Seagate/cortx/releases/tag/VA>`_, download and then uncompress the cortx-va-1.0.1.zip file that contains the virtual machine image.
+#. From `our release page <https://github.com/Seagate/cortx/releases/tag/VA>`_, download and then uncompress the `cortx-va-1.0.1.zip https://github.com/Seagate/cortx/releases/download/VA/cortx-va-1.0.2.zip`_ file that contains the virtual machine image.
```

Comment:
>download and then uncompress the `cortx-va-1.0.1.zip
It should be corrected to cortx-va-1.0.2.zip

Reply:
Thanks @mukul-seagate11 I'm not sure how I missed that. Thanks for catching that.